src/components/DataView/example/DataViewKitchenSink.tsx

```diff
@@ -172,7 +176,8 @@ const filters = [
 		args: {
 			getOptions: categoriesHelper.getOptions.bind(categoriesHelper),
 			getSelected: categoriesHelper.getSelected.bind(categoriesHelper),
-			comparisons: ["in", "not_in", "all", "exists", "not_exists"]
+			comparisons: ["in", "not_in", "all", "exists", "not_exists"],
+			placeholder: "Keyword..."

```

Comment:
For all of these where you added the placeholder, please take a look at the other comments... I feel if we keep the i18n constants + the ternary conditional on the components then there's no need to keep these here.